spec/integration/ffmpeg_adapter_spec.rb

```diff
@@ -112,6 +112,52 @@ def find_encode(id)
       end
     end
 
+    context "input file format does not match extension" do
+      let(:improper_format_file) { "file://" + Rails.root.join('..', 'spec', 'fixtures', 'file_without_metadata.mp4').to_s }
+      let(:improper_format_job) { ActiveEncode::Base.create(improper_format_file, outputs: [{ label: "low", ffmpeg_opt: "-s 640x480", extension: 'mp4' }]) }
+
+      it "returns the encode with correct error" do
+        expect(improper_format_job.errors).to include("Error inspecting input: #{improper_format_file}")
+        expect(improper_format_job.percent_complete).to be 1
+      end
+    end
+
+    context "input file with missing metadata" do
+      let(:file_without_metadata) { "file://" + Rails.root.join('..', 'spec', 'fixtures', 'file_without_metadata.webm').to_s }
+      let!(:create_without_metadata_job) { ActiveEncode::Base.create(file_without_metadata, outputs: [{ label: "low", ffmpeg_opt: "-s 640x480", extension: 'mp4' }]) }
+      let(:find_without_metadata_job) { ActiveEncode::Base.find create_without_metadata_job.id }
```

Comment:
Could you add a couple tests to show that these encode job objects have the correct duration?